makelib/uptest.mk

```diff
@@ -72,12 +88,18 @@ render: $(CROSSPLANE_CLI) ${YQ}
 			if [[ "$$COMPOSITION" == "null" || "$$FUNCTION" == "null" ]]; then \
 				continue; \
 			fi; \
+			LANG=$$(basename "$$(dirname "$$file")"); \
```

Comment:
That looks like not obvious assumption/convention. We need to document it explicitly

Reply:
Yes you're right, it's about having separate files, no need to bring assumption about language into picture :)